colossalai/fx/profiler/__init__.py

```diff
@@ -1,11 +1,12 @@
-from ... import META_COMPATIBILITY
-if META_COMPATIBILITY:
+from .._compatibility import check_meta_compatibility
+
+if check_meta_compatibility():
```

Comment:
Now I use the `check_meta_compatibility` API to check the `META_COMPATIBILITY`. Should have the same effect but be more user-friendly.